### ISSUE TITLE
Questions about SLIC

### ISSUE THREAD

**Issue by juanshu30** [pyg-team/pytorch_geometric#3158]
## ❓ Questions & Help

Hi,

I run the following code in the instructions but the dataset I got seems not to be superpixel images, I checked the train_data and it still returns a (60000, 28,28) tensor. I am wondering how can I get the superpixel images, thanks!

from torchvision.datasets import MNIST
import torchvision.transforms as T
from torch_geometric.transforms import ToSLIC

transform = T.Compose([T.ToTensor(), ToSLIC(n_segments=75)])
dataset = MNIST('/tmp/MNIST', download=True, transform=transform)

**Comment by rusty1s** [comment]
I sadly cannot reproduce this:
```python
from torchvision.datasets import MNIST
import torchvision.transforms as T
from torch_geometric.transforms import ToSLIC

transform = T.Compose([T.ToTensor(), ToSLIC(n_segments=75)])
dataset = MNIST('/tmp/MNIST', download=True, transform=transform)
print(dataset[0])
```
will print out
```
(Data(x=[81, 1], pos=[81, 2]), 5)
```
which holds a PyG data object of superpixels, and its corresponding label.

**Comment by juanshu30** [comment]
> I sadly cannot reproduce this:
> 
> ```python
> from torchvision.datasets import MNIST
> import torchvision.transforms as T
> from torch_geometric.transforms import ToSLIC
> 
> transform = T.Compose([T.ToTensor(), ToSLIC(n_segments=75)])
> dataset = MNIST('/tmp/MNIST', download=True, transform=transform)
> print(dataset[0])
> ```
> 
> will print out
> 
> ```
> (Data(x=[81, 1], pos=[81, 2]), 5)
> ```
> 
> which holds a PyG data object of superpixels, and its corresponding label.

I have tried this code and when I run print(dataset[0]), it gives the error 
"slic() got an unexpected keyword argument 'start_label'"
I am using the latest version of pyG and  cannot figure out why this happens.

**Comment by levulinh** [comment]
> I have tried this code and when I run print(dataset[0]), it gives the error
> "slic() got an unexpected keyword argument 'start_label'"
> I am using the latest version of pyG and cannot figure out why this happens.

Well, since PyG uses scikit-image for slic, you should also check the vesion of skimage on your environment
